docs/user/tutorials/01-40-expose-workload/README.md

```diff
@@ -2,14 +2,23 @@
 Browse the API Gateway tutorials to learn how to expose workloads. The tutorials are available in two versions: one uses the APIRule resource in version `v2alpha1` and the other uses the APIRule resource in version `v1beta1`. 
```

Comment:
ugh in three versions instead of two? v2 -> main stable version, v1beta1 -> deprecated, scheduled for removal, v2alpha1 -> testing short living version